### ISSUE TITLE
Add to F-Droid

### ISSUE THREAD

**Issue by nicolasmaia** [djuretic/praktika-vortaro#1]
Let's have this on F-Droid! :)

ping @rudloff

**Comment by Rudloff** [comment]
Hello,
If I understand correctly, the build needs a database generated from another repository. This will make it hard to package for F-Droid.
Would it be possible to automate this process?

**Comment by djuretic** [comment]
Yes, I think that it is possible, but it might take me a while to automate it :S

**Comment by djuretic** [comment]
@nicolasmaia better late than never: https://f-droid.org/en/packages/com.esperantajvortaroj.app/
